Dockerfile

```diff
@@ -1,5 +1,13 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
+FROM golang:1.16 as builder
+
+COPY . /asset-relocation-tool-for-kubernetes/
+ENV GOPATH=
+ENV PATH="${PATH}:/root/go/bin"
+WORKDIR /asset-relocation-tool-for-kubernetes/
+RUN make test && make build
```

Comment:
This needs to have `VERSION` set as a build arg, otherwise the LDFLAGS set by the makefile will set the version to `dev`

Reply:
Will fix

Reply:
Done:

~~~bash
$ docker build --build-arg VERSION=test-version . -t relok8s-test-version
$ docker run -it relok8s-test-version version
relok8s version: test-version
~~~

Reply:
> > Also, we'll need to make a change to the Concourse pipeline (while we're still using it) so that it's building the images properly.
> 
> The concourse test passed here, not sure what fix we need. Can you elaborate @petewall ?

Sorry, I should have been clearer. Since we're building the CLI inside docker, and now need to pass the version arg to the build step.

We'll need to adjust how the pipeline builds the production docker image. Currently, it takes the built binary and passes it to the docker build step.